.travis.yml

```diff
@@ -5,7 +5,7 @@ matrix:
     - compiler: clang
       env:
         - CXX_COMPILER='clang++-3.9'
-        - PYTHON_VER='2.7'
+        - PYTHON_VER='3.5'
```

Comment:
correct (i've since realized the duplicate). i'm dropping the duplicate, then leaving it be. sound right, or do you really want a case where py27/py35 is the sole change in build conditions?